webtau-feature-testing/src/test/groovy/org/testingisdocumenting/webtau/featuretesting/WebTauRestFeaturesTest.groovy

```diff
@@ -147,6 +147,11 @@ class WebTauRestFeaturesTest {
         runCli('springboot/listMatchByKey.groovy', 'springboot/webtau.cfg.groovy', "--url=$customersBaseUrl")
     }
 
+    @Test
+    void "proxy config"() {
+        runCli("proxy/passingProxy.groovy", "proxy/webtau.proxy.cfg.groovy", "--url=${testRunner.testServer.uri}")
```

Comment:
How does this actually work?  Any tests after this one will have those system properties set so there is a possibility they might actually run with a proxy.  I think this goes back to my earlier point around the lazy initialisation of the proxy and maybe we're actually relying on that here.